NEXTCLOUD.cmake

```diff
@@ -1,6 +1,6 @@
-set( APPLICATION_NAME       "Nextcloud" )
-set( APPLICATION_SHORTNAME  "Nextcloud" )
-set( APPLICATION_EXECUTABLE "nextcloud" )
+set( APPLICATION_NAME       "NextcloudDev" )
+set( APPLICATION_SHORTNAME  "NextDev" )
+set( APPLICATION_EXECUTABLE "nextclouddev" )
```

Comment:
I think these changes are not meant to be included, right?

Reply:
yes